docs/cudf/source/user_guide/10min.ipynb

```diff
@@ -115,7 +116,7 @@
    "source": [
     "ds = dask_cudf.from_cudf(s, npartitions=2)\n",
     "# Note the call to head here to show the first few entries, unlike\n",
-    "# cuDF objects, dask-cuDF objects do not have a printing\n",
+    "# cuDF objects, Dask-cuDF objects do not have a printing\n",
```

Comment:
I don't want to annoy you with this @rjzamora , but just wanted to point out in case you missed that you left a few `Dask-cuDF` entries (with hyphen) and you wanted to explicitly move away from it.

Reply:
I think it's fine to hyphenate Dask cuDF when it is used as an adjective.